docset/winserver2022-ps/networkcontroller/Get-NetworkControllerLoadBalancerBackendAddressPool.md

```diff
@@ -127,9 +127,13 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## INPUTS
 
+### System.String[]
+
 ## OUTPUTS
 
-### 
+### System.Object
+
+### System.Object#https://localhost/
```

Comment:
can you check this line?

Reply:
I removed this line in the new cmdlets per product team input. I'll remove them from these revised ones in bulk.